examples/workdir/inventory/hosts.yml

```diff
@@ -1,4 +1,8 @@
 ---
+localhost:
+  vars:
+   ansible_python_interpreter: /usr/bin/python3
+
 caasp-admin:
   vars:
     ansible_user: root
```

Comment:
The target nodes however will be tested against <python version of the deployer>. Assuming a venv is used for example, this will fail, and then cause a fallback to known python, which might be symlinked to python2.

For that, we should have ``ansible_python_interpreter: /usr/bin/python3`` for those groups.
While this seems to be done when generating the real inventory, I think we should also do it with the example.

Because there seems to be a lot of repetitions, I would do the following for caasp-admin:
```
caasp-admin:
  vars: &connectionsettings
    ansible_user: "root"
    ansible_python_interpreter: "/usr/bin/python3"
```

and then, for all the other groups, use:
```
vars: *connectionsettings
```
It seems it's only wanted for some groups, but you get the idea.

Reply:
sed caasp-admin with the appropriate host who needs python3.